app/models/tenant.rb

```diff
@@ -0,0 +1,8 @@
+class Tenant < ApplicationRecord
+  has_many :users
+  validates :name, presence: true, length: { minimum: 3, maximum: 64 }
+  VALID_WEBSITE_REGEX = /\A[-a-z0-9_]+\.+[-a-z0-9\/]+\z/i
```

Comment:
Try to follow the next order of declaration in model class:

 * include / require
 * CONSTANTS
 * attr readers/writers/accessors 
 * custom / gem method direct invocation
 * file uploaders 
 * associations
 * nested attributes
 * scopes
 * validations
 * callbacks
 * class methods
 * instance methods 
 * private/protected methods